website/profile/views.py

```diff
@@ -353,9 +354,10 @@ def user_account_password(auth, **kwargs):
         user.change_password(old_password, new_password, confirm_password)
         user.save()
     except ChangePasswordError as error:
-        push_status_message('<br />'.join(error.messages) + '.', kind='warning')
+        for m in error.messages:
+            push_status_message(m, kind='warning', trust=False)
```

Comment:
Multiple individual status messages seems like it is more in line with behavior elsewhere on the site. (eg registrations with public/private addons of various types) Each individual problem with the form input is given its own space.

The reason for the HTML was to display them cleanly on separate lines. I'm very reticent to move responsibility for HTML generation further into the backend (eg if we added more markupsafe escaping for all variables before sending a user-generated string).

`push_status_message` could be used to send a separate CSS class instead using some [whitespace tricks](http://stackoverflow.com/questions/2703601/how-to-give-line-break-from-css-without-using-br)... but this could introduce display/browser anomalies of its own, and it still leaves the issue of display concerns being folded into the backend, plus special casing. We can weigh UI options tomorrow when I'm back in the office.